src/compas/geometry/shapes/cone.py

```diff
@@ -96,7 +96,7 @@ def circle(self):
 
     @circle.setter
     def circle(self, circle):
-        self._circle = circle
+        self._circle = Circle(*circle)
```

Comment:
while fixing this bug, i noticed that the cone does not have a bottom face. should it?